core/jazz_services/components/utils.js

```diff
@@ -162,7 +163,26 @@ var getUpdateData = function (update_data) {
             } else {
                 input_data[field] = [];
             }
-        } else {
+        }
+        else if (update_data[field] && update_data[field].constructor === Object) {
```

Comment:
move this to the line above "} else if{...."

Reply:
done